pcap.c

```diff
@@ -4211,6 +4211,14 @@ pcap_read_dead(pcap_t *p, int cnt _U_, pcap_handler callback _U_,
 	return (-1);
 }
 
+static int
+pcap_breakloop_dead(pcap_t *p)
+{
+	snprintf(p->errbuf, PCAP_ERRBUF_SIZE,
```

Comment:
`pcap_breakloop()` implies there's a loop inside pcap to be broken out of - i.e., that it's possible to capture packets from, or read packets from, the `pcap_t` in question.

It's not possible to capture packets from, or read packets from, a dead `pcap_t`, so `pcap_breakloop()` would be a no-op.  In addition, as `pcap_breakloop()` does not return a value, there's no way to indicate failure.  Therefore, this function should just return and do nothing.

Reply:
thanks. 

I have fixed that in the next commit.